### ISSUE TITLE
Lexing error with Exclamation marks (!)

### ISSUE THREAD

**Issue by mccartney** [bertramdev/hcl4j#30]
Context: Started using hcl4j to parse yet another Terraform codebase in my org and I encounter another batch of errors. Thus I thought I should report them.

The following seems to be a legit Terraform code:
```
locals {
  maybe = true
  perhaps = ! local.maybe
}
```
while it results in the error:
```
    com.bertramlabs.plugins.hcl4j.HCLParserException: Illegal character <(!) - state: 14> found on line: 4 col: 13
        at com.bertramlabs.plugins.hcl4j.HCLLexer.yylex(HCLLexer.java:1368)
```

**Comment by davydotcom** [comment]
good catch! Will fix , sorry for delay did not see this notification.

**Comment by davydotcom** [comment]
fixed a bit ago but tested in 0.7.5
